packages/byods/package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "@webex/byods",
+  "files": [
+    "dist"
+  ],
+  "contributors": [],
+  "main": "dist/module/index.js",
+  "module": "dist/module/index.js",
+  "types": "dist/types/index.d.ts",
+  "license": "MIT",
+  "author": "devsupport@webex.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webex/webex-js-sdk.git",
+    "directory": "packages/byods"
+  },
+  "engines": {
+    "node": ">=20.x"
+  },
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "tsc",
+    "build:src": "tsc",
+    "clean": "rimraf dist",
+    "test:unit": "jest --config=jest.config.js --runInBand",
+    "test:style": "eslint 'src/**/*.ts'",
+    "fix:lint": "eslint 'src/**/*.ts' --fix",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "build:docs": "typedoc --out ../../docs/byods",
+    "docs": "typedoc --emit none",
+    "deploy:npm": "yarn npm publish"
```

Comment:
1. Is it possible to leverage the webex-legacy-tool's build command for this to maintain uniformity in the repo?

2. Since we dont have a jest.config can we remove this command, we have a story for adding tests in the package, we can add this command then. Also lets try to use webex-legacy-tools for the test.

3. Do we need 2 types of lint fixer.